src/_includes/docs/install/reqs/macos/zsh-config.md

```diff
@@ -0,0 +1,51 @@
+{% assign target = include.target %}
+
+<details markdown="1">
+<summary><strong>To verify your shell configuration, expand this section</strong></summary>
+
+Like most UNIX-like operating system, macOS can support multiple shells,
+like `bash`, `zsh`, and `sh`.
+
+As of the October 2019 release of macOS Catalina (macOS 10.15),
+Z shell or `zsh` is the default shell for macOS.
+Apple made this change to due to licensing changes for `bash`.
+
+#### Check and set `zsh` as default
+{:.no_toc}
+
+1. To verify `zsh` has been installed and set as
+   the default macOS shell, run the following commands.
+
+    ```terminal
+    $ which zsh; dscl . -read ~/ UserShell
```

Comment:
This can be simplified to:

```suggestion
$ echo $0
/bin/zsh
```

Reply:
No, it can't. The second part tells you what your default shell is. The `echo $0` only tells you what shell is running in your current terminal.

Reply:
I think it's safe to assume the user is running their default shell.

Reply:
We can't make too many assumptions. This is one that I think would be important to explain.

Reply:
I don't think average Flutter developer is concerned about this honestly, and this isn't even documenting something about Flutter. I would vote that we keep our docs as general as possible since not every developer even uses Zsh, and they don't need to use Zsh to use Flutter, Cocoapods, or anything else.

Reply:
They need to know which shell they're running to properly install all of these apps because of where they need to set the permanent PATH. That's why I'm calling this out: They should use the macOS default, `zsh`, and the default file for setting environment variables, `.zshenv`.

Reply:
I understand your point. I believe this can answer questions that may be raised later. I stand by the change. If the community submits issues to the contrary, we can revisit then.